src/main/java/calculator/PositiveNumbers.java

```diff
@@ -0,0 +1,26 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PositiveNumbers {
+
+    private final List<PositiveNumber> positiveNumbers;
+
+    public PositiveNumbers(List<PositiveNumber> positiveNumbers) {
+        this.positiveNumbers = positiveNumbers;
+    }
+
+    public PositiveNumbers(String[] numbers) {
+        this.positiveNumbers = Arrays.stream(numbers)
+            .map(PositiveNumber::new)
+            .collect(Collectors.toList());
+    }
+
+    public int sumPositiveNumber() {
+        return positiveNumbers.stream()
+            .mapToInt(PositiveNumber::getValue)
+            .sum();
+    }
```

Comment:
```suggestion
    public int sum() {
        return positiveNumbers.stream()
                .reduce(ZERO, PositiveNumber::sum)
                .getValue();
    }
```
- 값을 꺼내서 비교하지 않고, 메시지를 보낼 수 있지 않을까요